github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
there's `status`, which i think you have a filter for. I don't think it's unreasonable to print that out though